.github/workflows/ci.yml

```diff
@@ -224,7 +207,7 @@ jobs:
     name: Notifier
     needs:
       - defaults  # so we can access `inputs.root-sbd` that could have defaulted to `inputs.model`
-      - version-tag  # implies all the spack.yaml-related checks have passed, has appropriate version for the prerelease build
+      - version  # implies all the spack.yaml-related checks have passed, has appropriate version for the prerelease build
```

Comment:
Comment message below still references `${{  needs.version-tag.outputs.release }}` and `${{ needs.version-tag.outputs.prerelease }}`.

Reply:
Done in https://github.com/ACCESS-NRI/build-cd/pull/101/commits/adefb3953ea1ddd9291a51f0ebd7f08cecf8f6be